.circleci/config.yml

```diff
@@ -164,21 +164,24 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - mfr-dependencies-{{ checksum "react/package.json" }}-0
+            - mfr-dependencies-{{ checksum "react/package.json" }}-4
       - run: cd react && npm install
       - save_cache:
           paths:
             - react/node_modules
-          key: mfr-dependencies-{{ checksum "react/package.json" }}-0
+          key: mfr-dependencies-{{ checksum "react/package.json" }}-4
       - run:
           name: Mayflower React Build
           command: cd react && npm run build
-      - run:
-          name: Mayflower React Linter
-          command: cd react && npm run lint
       - run:
           name: Mayflower React Build Storybook
           command: cd react && npm run build-storybook
+      - run:
+          name: Mayflower React Storybook Extract
+          command: cd react && npm run storybook:extract
+      - run:
+          name: Mayflower React Linter
+          command: cd react && npm run lint
```

Comment:
is it on purpose to order linting after build?

Reply:
Yes - the file generated by storybook extract needs to exist, else lint fails.